homeassistant/components/aqara/README.md

```diff
@@ -0,0 +1,4 @@
+# aqara-home-assistant
```

Comment:
This file seems like it's left over from this being a stand-alone custom component?
I think it should be removed.